### ISSUE TITLE
Metadata

### ISSUE THREAD

**Issue by pijulius** [ordinals/ord#2116]
Hi Guys,

I'm new to this so would like to bring up the metadata problem with ordinals on my own thinking. 

It seems there is no way to get the name, collection, creator or not to mention attributes of an inscription atm. I read some ideas about these things but let me bring up this as it would be already working, without any modification to anything.

### Simple Solution:
Append a json text to the end of the inscription content. 

**Advantages:** 
> Would work right away, would not brake any image viewers, or video viewers, even executables and package files should work without any change needed as they are all ignoring the additional data at the end. All existing wallets, websites and so on would show the images and content correctly just as right now and all new integration could use the json data from the end of the content to parse collection data, attributes, even a creator wallet that could be confirmed by signing a request and so be sure you talk to the owner of the collection.

> No need for additional protocols or other new implementation to websites or wallets or anything else, we would just need to agree to a JSON format to use and that would be all,

**Downside:**
> it wouldn't work with text and html content types but as far as I know all the rest of the files would work just fine.

So bottom line, what do you guys think about this? was it already talked about? was it already thrown away for some reason? please let me know what do you guys think.

My2c re Ordinals Metadata

**Comment by raphjaph** [comment]
We add metadata into the tags of the inscription envelope. I'm writing documentation on that so will be out soon.
